AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -271,15 +311,15 @@ - (void)createWithPartition:(NSString *)partition
     return;
   }
   [self performOperationForPartition:partition
-                          documentId:documentId
+                          documentId:@""
```

Comment:
Can you explain this change?

Reply:
bug fix, to create a doc, you don't need document ID

Reply:
Why not nil then?